teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt

```diff
@@ -135,7 +135,7 @@ class TeamscalePluginTest {
             .withProjectDir(temporaryFolder.root)
             .withPluginClasspath()
             .withArguments(runnerArgs)
-            .withGradleVersion("4.6")
+            .withGradleVersion("6.5")
```

Comment:
Why did you need to update the gradle version here?

4.6 is our minimal supported version and we run the tests against this version to ensure everything still works on this version.
When updating we should also adjust https://github.com/cqse/teamscale-jacoco-agent/blob/master/teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscalePlugin.kt#L50

Reply:
Ah I see. I was unable to run the tests on my machine since Gradle 4.6 seems to not run under Java 11, so I just updated the version to match what we use elsewhere. What's your opinion? Leave at 4.6 or bump to the first Gradle version that supports Java 11?

Reply:
Bump to Java 11 is fine for me. We are soon on 6.6.1 so 6.5 would be fine.

Reply:
Could you please update the version in TeamscalePlugin (see link above)?